gammapy/maps/region.py

```diff
@@ -46,16 +51,16 @@ class RegionGeom(Geom):
     _slice_spatial_axes = slice(0, 2)
     _slice_non_spatial_axes = slice(2, None)
     projection = "TAN"
-    binsz = 0.01
 
-    def __init__(self, region, axes=None, wcs=None):
+    def __init__(self, region, axes=None, wcs=None, binsz_wcs=0.01):
```

Comment:
Maybe add the unit for the default by using a string: `"0.01 deg"`

Reply:
`WcsGeom.create()` needs `binsz` to be a float though. I can add `binsz_wcs = Angle(binsz_wcs)` before calling it, but maybe it makes more sense to add that line inside `WcsGeom.create()`?

Reply:
Yes, please, I think we should support this in `WcsGeom.create()` as well.